games/fheroes2/Portfile

```diff
@@ -7,12 +7,6 @@ PortGroup       cmake 1.1
 PortGroup       legacysupport 1.1
 PortGroup       compiler_blacklist_versions 1.0
 
-# On <10.15 built-in libc++ has no support for std::filesystem
-legacysupport.newest_darwin_requires_legacy 18
-if {[string match *clang* ${configure.compiler}]} {
```

Comment:
The reason this is left here is because it breaks the build if GCC is forced. @barracuda156 does tests using it because the last stable working version of Clang for <10.6 is 11.0 and it's already pretty outdated.

Reply:
@kencu [said](https://github.com/macports/macports-ports/pull/18690#issuecomment-1552356314) that compiler check is not necessary. Are you sure that it is 100% necessary?

Reply:
@oleg-derevenetz it's just necessary for a non-standard environment.

Reply:
@aeiouaeiouaeiouaeiouaeiouaeiou OK, then I'll leave the compiler check in place.